Tests/Form/Type/FSi/FormTypeTest.php

```diff
@@ -55,15 +57,20 @@ public function setUp()
             'form_div_layout.html.twig',
             'Form/form_div_layout.html.twig'
         ], $twig);
-        $renderer = new TwigRenderer(
-            $rendererEngine,
-            $this->getMockBuilder('Symfony\Component\Security\Csrf\CsrfTokenManagerInterface')->getMock()
-        );
+        if (class_exists(FormRenderer::class)) {
```

Comment:
This class exists in earlier versions of symfony than 3.4, but should be used in this context only starting from version 3.4 of [symfony/twig-bridge](https://github.com/symfony/twig-bridge). I think that [this](https://github.com/jordisala1991/SonataAdminBundle/blob/38763f897bdf7ef697674db28f96de89ae47e5aa/src/Controller/HelperController.php#L562-L573) is the proper way of checking this.